packages/cli/src/main.ts

```diff
@@ -19,12 +19,19 @@ timingApi.mark("PRE_IMPORT_IMPERATIVE");
 import { IImperativeConfig, Imperative } from "@zowe/imperative";
 import { Constants } from "./Constants";
 import { inspect } from "util";
+import { homedir } from "os";
 
 // TODO(Kelosky): if we remove this, imperative fails to find config in package.json & we must debug this.
 const config: IImperativeConfig = {
-    configurationModule: __dirname + "/imperative"
+    configurationModule: "imperative.js"
```

Comment:
Curious why this was changed to a relative path. Having the absolute path could be useful in some cases.